overview/saml.mdx

```diff
@@ -15,6 +15,43 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+We're going to set up our PropelAuth Project to enable Acme Co to set up a SAML connection.
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML. You can also enable SAML on a per org basis by enabling the Can setup SAML option in the organization settings
```

Comment:
This is my fault, my naming here is bad... it's under Acme Co's settings page (e.g. manage > organizations > acme co > edit) as opposed to the organization settings page.